Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM fuzzers/afl:2.52
```

Comment:
Can we please add a package stage here?

Reply:
I can refactor the dockerfile. I assume you mean to use a distro package instead right, like FROM/ubuntu:20.04
Will do it shortly..

Reply:
No - I mean can we separate the Dockerfile into a build and package stage. Here's an example of another PR with a package stage:
https://github.com/evanrichter/watchman/blob/main/mayhem/Dockerfile
See here for more info: https://docs.docker.com/build/building/multi-stage/

Reply:
Done